javascript/javascript-import-and-require.md

```diff
@@ -0,0 +1,79 @@
+# JavaScript - Require & Import
+
+## Project Time
+
+About 45 Minutes
+
+- 30 minutes for the video and reference material
+- 10 minutes for Check for Understanding

```

Comment:
Please make all these points like `Check for Understanding: 10 min` as in the template.